src/majsoulrpa/_impl/browser.py

```diff
@@ -185,9 +185,13 @@ def __init__(
         mute_audio_off = None if headless else ["--mute-audio"]
 
         self._context_manager = sync_playwright()
-
+        
         self._browser = None
         if user_data_dir:
+            if isinstance(user_data_dir, str):
+                user_data_dir = Path(user_data_dir)
+            if not user_data_dir.is_absolute():
+                user_data_dir = Path.cwd() / user_data_dir

```

Comment:
1. Use `user_data_dir.resolve()` instead.
2. What do you think about the idea of use `user_data_dir.resolve()` regardless of the result of `user_data_dir.is_absolute()`?

Example:
```python
        self._browser = None
        if user_data_dir:
            if isinstance(user_data_dir, str):
                user_data_dir = Path(user_data_dir)
            user_data_dir = user_data_dir.resolve()

            self._context = self._context_manager.start().chromium.launch_persistent_context(  # noqa: E501
```

Reply:
Did not know that resolve() exist, I think `user_data_dir.resolve()` would be better.